src/ArrayExpressionLanguageProvider.php

```diff
@@ -17,6 +17,7 @@ public function getFunctions(): array
             ExpressionFunction::fromPhp('count', 'count'),
             ExpressionFunction::fromPhp('array_combine', 'combine'),
             ExpressionFunction::fromPhp('iterator_to_array', 'iterableToArray'),
+            ExpressionFunction::fromPhp('array_filter'),
```

Comment:
```suggestion
            ExpressionFunction::fromPhp('array_filter', 'filter'),
```

Il manque de la doc sur cette fonction et des callback utilisables